db/schema.rb

```diff
@@ -217,6 +218,8 @@
     t.string 'cataloguer'
     t.string 'user_name'
     t.string 'user_phone'
+    t.string 'created_by'
```

Comment:
here also, `created_by` is added and I'm thinking why this might be the case 🤔 maybe I'm missing something here.